### PR TITLE
Quick review of the remember me article

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

File: cookbook/security/remember_me.rst [hunk 1]
```diff
@@ -167,15 +173,18 @@ The Security component provides an easy way to do this. In addition to roles
 explicitly assigned to them, users are automatically given one of the following
 roles depending on how they are authenticated:
 
-* ``IS_AUTHENTICATED_ANONYMOUSLY`` - automatically assigned to a user who is
-  in a firewall protected part of the site but who has not actually logged in.
-  This is only possible if anonymous access has been allowed.
+``IS_AUTHENTICATED_ANONYMOUSLY``
+  Automatically assigned to a user who is in a firewall protected part of the
+  site but who has not actually logged in. This is only possible if anonymous
+  access has been allowed.
```
Comment (xabbuh) [inline cookbook/security/remember_me.rst]: Doesn't the text need to be indented?

File: cookbook/security/remember_me.rst [hunk 2]
```diff
@@ -25,15 +25,21 @@ the session lasts using a cookie with the ``remember_me`` firewall option:
...
-                <remember-me
-                    key      = "%secret%"
-                    lifetime = "604800" <!-- 1 week in seconds -->
-                    path     = "/"
-                />
-            </firewall>
-        </config>
+        <?xml version="1.0" encoding="utf-8" ?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:srv="http://symfony.com/schema/dic/services">
```
Comment (xabbuh) [inline cookbook/security/remember_me.rst]: Shouldn't we add the XSD reference for the http://symfony.com/schema/dic/services namespace?

```xml
<srv:container xmlns="http://symfony.com/schema/dic/security"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:srv="http://symfony.com/schema/dic/services"
    xsi:schemaLocation="http://symfony.com/schema/dic/services
        http://symfony.com/schema/dic/services/services-1.0.xsd">
```

File: cookbook/security/remember_me.rst [hunk 3]
```diff
@@ -167,15 +173,18 @@ The Security component provides an easy way to do this. In addition to roles
...
-  This is only possible if anonymous access has been allowed.
+``IS_AUTHENTICATED_ANONYMOUSLY``
+  Automatically assigned to a user who is in a firewall protected part of the
+  site but who has not actually logged in. This is only possible if anonymous
+  access has been allowed.
 
-* ``IS_AUTHENTICATED_REMEMBERED`` - automatically assigned to a user who
-  was authenticated via a remember me cookie.
+``IS_AUTHENTICATED_REMEMBERED``
+  Automatically assigned to a user who was authenticated via a remember me
```
Comment (javiereguiluz) [inline cookbook/security/remember_me.rst]: As pointed previously by @xabbuh, the description should be indented if we want to display it as a definition list.

File: cookbook/security/remember_me.rst (hunk 2 above)
Comment (WouterJ) [inline cookbook/security/remember_me.rst]: YEah, but I can never remember those URLs...

**Comment by xabbuh** [comment]
:+1:

**Pull request merged**